person.rb

```diff
@@ -27,4 +28,8 @@ def correct_name
   def of_age?
     @age >= 18
   end
+
+  def add_rental(rental)
+    @rental << rental
+  end
```

Comment:
- [ ] Kindly follow the same implementation of creating a rental like what you did in the `Book` class.

Reply:
@ShadyShawkat !
Thank you for reviewing my code I will change it.